src/components/DisplayNames/index.js

```diff
@@ -100,7 +100,7 @@ class DisplayNames extends PureComponent {
                             >
                                 {/*  // We need to get the refs to all the names which will be used to correct
                                     the horizontal position of the tooltip */}
-                                <Text ref={el => this.childRefs[index] = el}>
+                                <Text ref={el => this.childRefs[index] = el} style={[...this.props.textStyles, styles.pRelative]}>
```

Comment:
Do we need styles.pRelative here?

Reply:
Yeah I'm not sure why we need relative positioning there since it works just fine without it. I just added it since it was supposed to [affect the text](https://github.com/Expensify/App/blob/srikar-chatNameColor/src/components/DisplayNames/index.js#L86) but I can take it out. It doesn't affect any platform but I was just scared that there might be some edge case for the reason it was added.

Reply:
Let's remove `pRelative` from line 103 and leave it on line [86](https://github.com/Expensify/App/blob/18490e516d3c5aa69bc6c7d00b1b81132072d3e2/src/components/DisplayNames/index.js#L86) just to be safe.

Reply:
sure, that sounds good, the text object will be positioned accordingly anyway in that case